### PR TITLE
Edge AI Update (And more!)

### PR DESCRIPTION
**Pull request by therealDLondon**
## About the PR
Changes the Logi conveyor (adds security to the flaps)
Adds more evac screens
Replaces chemmaster in botany with bioregen machine
Decal changes
Changes the redundant security hallway (Expands changing room)
Changes firing range door to glass door
Adds warden's hardsuit
Minor lighting changes
Adds command AI tube
Updates AI sat
Adds robotics lab
Remodels LO's Office
Adds salvage recycler
Adjusts singulo/tesla containment
Adds tesla parts
Adds AI and Robotisist roles

## Why / Balance
Why not

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
I dont think I need one this time

### CODE REVIEW

File: Resources/Maps/edge.yml [hunk 1]
```diff
@@ -12993,6 +13011,25 @@ entities:
     - type: Transform
       pos: -53.470978,3.6152563
       parent: 2
+    - type: GroupExamine
```
Comment (IamVelcroboy) [inline Resources/Maps/edge.yml]: Looks like this box got borked. Will need to replace it or delete troll comps

File: Resources/Maps/edge.yml [hunk 2]
```diff
@@ -89688,14 +89786,6 @@ entities:
     - type: Transform
       pos: 8.300806,20.221859
       parent: 2
-  - uid: 16841
```
Comment (IamVelcroboy) [inline Resources/Maps/edge.yml:89691]: No more special warden plant?

Comment (therealDLondon) [reply Resources/Maps/edge.yml:89691]: I had to make the hardsuit locker fit

File: Resources/Maps/edge.yml (hunk 1 above)
Comment (IamVelcroboy) [reply Resources/Maps/edge.yml]: This through line 13032 need to be deleted still.

**Comment by IamVelcroboy** [comment]
AI transfer tube room needs power and a camera 
image
image

**Comment by IamVelcroboy** [comment]
Camera in here 
image

**Comment by IamVelcroboy** [comment]
probably a camera here too 
image

**Comment by IamVelcroboy** [comment]
Redundant solar controls, make one a power monitor 
image

**Comment by IamVelcroboy** [comment]
Overall casing consistency with cameras(in the future I'm going to favor upper case) and one unnamed. 
image
image

**Comment by IamVelcroboy** [comment]
AI sat has distro but no waste

**Comment by IamVelcroboy** [comment]
Concludes review. I'll merge after those fixes.

**Review by IamVelcroboy — APPROVED** [review]
Ready for merge

**Pull request merged**